.htaccess

```diff
@@ -1,3 +1,4 @@
+ErrorDocument 404 /views/pages/404.html
```

Comment:
I added this line.
(THERE ARE YOU HAPPY GITHUB!!!! ARE YOU HAPPY THAT I MADE A COMMENT ON THE SINGLE LINE THAT I CHANGED!!!!!!!! WILL YOU MAKE THE NOTIFICATION GO AWAY FROM THE STUPID BOTTOM OF THE STUPID PAGE NOW GITHUB!!!!!!!!!!!!!!!!!!! ANSWER ME!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!)